operator/roles/ansible-service-broker/tasks/main.yml

```diff
@@ -37,6 +37,17 @@
 - name:  Validate pre-requisites for {{ state }}
   import_tasks: "validate_{{ state }}.yml"
 
+- name: Set broker admin cluster rolebinding state={{ state }}
+  k8s:
+    state: "{{ state }}"
+    definition: "{{ lookup('template', 'broker-admin.clusterrolebinding.yaml.j2') | from_yaml }}"
+  register: crb_status
+  ignore_errors: yes
+
+- fail:
+    msg: "Operator does not have sufficient permissions to create RBAC policy for the broker, please check the documentation."
+  when: crb_status.status is defined and crb_status.status == 403
```

Comment:
I would also make sure that we fail if crb_status is 50X errors. 

I think the best way to do this is `crb_status.status != 20X`.

Reply:
I agree that the playbook will fail if it hits 50X errors, but then again 50X errors are not related to RBAC right? So it would still be inappropriate error reporting.